UnityProject/Assets/Scripts/Database/ServerData.cs

```diff
@@ -96,8 +96,10 @@ void OnDisable()
 		void Update()
 		{
 			if (config != null)
-			{
-				MonitorServerStatus();
+				if (!string.IsNullOrEmpty(config.HubUser))
```

Comment:
In order to replicate previous behavior, it should probably check the same things.
```suggestion
				if (!string.IsNullOrEmpty(config.HubUser) && !string.IsNullOrEmpty(config.HubPass))
```

Reply:
I think I messed up the formatting.

Reply:
It looks like it does not have enough curly braces.